api/src/main/java/org/pmiops/workbench/exfiltration/jirahandler/EgressInternalJiraHandler.java

```diff
@@ -0,0 +1,116 @@
+package org.pmiops.workbench.exfiltration.jirahandler;
+
+import java.time.Clock;
+import java.util.Optional;
+import java.util.logging.Logger;
+import java.util.stream.Stream;
+import javax.inject.Provider;
+import org.pmiops.workbench.config.WorkbenchConfig;
+import org.pmiops.workbench.db.model.DbEgressEvent;
+import org.pmiops.workbench.db.model.DbUser;
+import org.pmiops.workbench.db.model.DbWorkspace;
+import org.pmiops.workbench.exfiltration.EgressRemediationAction;
+import org.pmiops.workbench.jira.ApiException;
+import org.pmiops.workbench.jira.JiraContent;
+import org.pmiops.workbench.jira.JiraService;
+import org.pmiops.workbench.jira.model.AtlassianContent;
+import org.pmiops.workbench.jira.model.SearchResults;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+@Service("internalJiraHandler")
+public class EgressInternalJiraHandler extends EgressJiraHandler {
```

Comment:
What does `Internal` mean here?  To me, this just another kind of Jira ticket about file name too large. We can not reuse the same service to other type 'internal' jira ticket. I think we may 
1: Make this name to be more expliciate
2: Refactor this class that can be handle "internal" type of event.

Reply:
Internal to me is because th eevent is not coming from Sumologic, but rather from an "internal" RW job, I'll think of another name though.